### ISSUE TITLE
Income Limits: Update breadcrumbs and title tag to match guidance

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18862]
## Background
The guidance for breadcrumbs and `<title>` tags has been updated and formalized through content and accessibility. Now that it has been completed, there are some small adjustments to Income limits that should be made to bring this in alignment.

## Description
1. `<title>` tag updates
   - Currently, for each screen the `<title>` is `{H1} | Income Limits | Veterans Affairs`
   - To align with the new guidance, we are going to drop the H1 of each screen and use just the H1 of the flow itself
   - The `<title>` tag for each screen will be `Income limits and your VA health care | Veterans Affairs` for each screen of the flow.
2. Breadcrumb updates
   - All crumbs within the Income limits flow should be: `VA.gov home > VA Health care` > Income limits and your VA health care`
   - The specific updates are:
     - Change the second crumb to be "VA Health care" to match the H1 of the page this leads to. Make sure the capitalization is correct (lower case C on care)
     - Change the third/last crumb to be "Income limits and your VA health care" to match the H1 of the intro page of the form flow

### Additional info
Functionality the last crumb should also be going to the intro page but the v3 component does not allow for this so for now this will still be an anchor link to the current page content.

## Acceptance Criteria
- [x] The `<title>` tag for each screen in the form flow is `Income limits and your VA health care | Veterans Affairs` 
- [x] The first breadcrumb for each screen is `VA.gov home`
- [x] The second breadcrumb for each screen is `VA Health care` (edited by Randi to capitalize the "H" in `Health` per the description above)
- [x] The last breadcrumb for each screen is `Income limits and your VA health care`
- [x] Requires accessibility review

**Comment by randimays** [comment]
Pulling in as stretch for sprint 13.

**Comment by randimays** [comment]
Successfully validated in production. Closing as complete.
